templates/Layout/SimpleStyleguideController.ss

```diff
@@ -0,0 +1,45 @@
+
+
+<% if $HeaderElements %>$HeaderElements<% end_if %>
+<div class="container">
+
+        <div class="row">
+            <section class="<% if $Children %>col-lg-12<% else %>col-lg-8 offset-lg-2<% end_if %>">
+                <header class="page-header">
+                    $Breadcrumbs
+                    <h1>$Title</h1>
+                </header>
+            </section>
+        </div>
+
+    <div class="row">
+        <section class="col-lg-8<% if not $Children %> offset-lg-2<% end_if %>">
+            <div class="styleguide">
+
+                $Message
+
+                <% include SimpleStyleguideNavigation %>
```

Comment:
This display an inpage navigation, I would have preferred to display it instead of the `SidebarNav`, but that's currently broken.

If the intention is to fix that, then I would suggest moving the navigation there.